Overview.src.html

```diff
@@ -592,11 +591,19 @@ <h4 id=terminology-headers>Headers</h4>
 <dfn title=concept-request-destination>destination</dfn>, which is one of
 the empty string,
 "<code title>document</code>",
+"<code title>embed</code>",
+"<code title>font</code>",
+"<code title>image</code>",
+"<code title>object</code>",
+"<code title>manifest</code>",
+"<code title>media</code>",
+"<code title>report</code>",
 "<code title>serviceworker</code>",
+"<code title>script</code>",
 "<code title>sharedworker</code>",
-"<code title>subresource</code>",
-"<code title>unknown</code>", and
-"<code title>worker</code>".
+"<code title>style</code>",
+"<code title>worker</code>", and
+"<code title>xslt</code>".
```

Comment:
We should add that the initial value is the empty string here.  (I can fix this while landing, just making a note here.)